### PR TITLE
Documented scripts used in the baking workflows, the workflows and ho…

### PR DESCRIPTION
**Pull request by edwoodward**
…w to install neb and cnx-recipes on ubuntu

### CODE REVIEW

File: script/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+ * `sudo apt-get install libxml2-utils`
+
+**Install xsltproc**
+ * `sudo apt-get install xsltproc`
+
+**Install virtualenv**
+ * `sudo pip install virtualenv`
+
+**Install Neb**
+ * `pip install nebuchadnezzar`
```
Comment (karenc) [inline script/README.md]: This should be in `./script/setup` so no need to have a separate command to do it.

File: script/README.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+**Clone cnx-recipes**
+ * `git clone https://github.com/openstax/cnx-recipes.git`
+
+**Install yarn**
+ * `curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -`
+ * `echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list`
+* `sudo apt-get update && sudo apt-get install yarn`
+
+**Install nvm**
+* `sudo apt-get install npm`
```
Comment (karenc) [inline script/README.md]: Is it meant to be nvm or npm?

File: script/README.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+6. Test the baked results in the browser
+
+
+PDF Style Development Workflow
+------------------------------
+
+**NOTE**: This workflow assumes the recipe workflow has been completed
+
+1. Move the baked HTML file to the location of the book resources
+    * `cp ./data/intro-business-baked.xhtml ./data/intro-business-baked/contents`
```
[the remark this answers is not in the document]
Comment (brittany-johnson) [reply script/README.md]: `./data/intro-business-baked` is the file output from running `./script/bake-book intro-business`

[the remark this answers is not in the document]
Comment (karenc) [reply script/README.md]: @brittany-johnson I thought `./script/bake-book intro-business` only creates `./data/intro-business-baked.xhtml` (old) or `./data/intro-business/collection.baked.xhtml` (new).  I'm not aware of it creating a directory called `./data/intro-business-baked`, may be I missed something though :sweat_smile:

[the remark this answers is not in the document]
Comment (edwoodward) [reply script/README.md]: @karenc could be that the old ssh fetch created that directory?

File: script/README.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+2.  Update path to the style CSS file in the baked HTML
+3.  Build the styles
+    * `./script/build-styles intro-business`
+4.  Test the styles by viewing the baked HTML in the browser
+5.  Convert the baked HTML and style into a PDF
+    * `prince -s styles/output/intro-business.css ./data/intro-business-baked/contents/intro-business-baked.xhtml -o bakedEcon.pdf`
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply script/README.md]: Change has been made.

File: script/README.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+Recipe Development Workflow
+----------------------------
+
+1. Fetch the data 
+   * `./script/fetch-html intro-business`
+2. Build the recipe
+    *  `./script/build-recipes intro-business`
+3. Open the raw HTML file (Book HTML file) 
+   * `atom ./data/intro-business-raw.xhtml`
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply script/README.md]: Change has been made.

File: script/README.md (hunk 1 above)
Comment (edwoodward) [reply script/README.md]: @karenc I removed it from the instructions

File: script/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (karenc) [reply script/README.md]: @edwoodward The old ssh fetch creates `./data/intro-business-raw.xhtml` and downloads any resources in `./resources`.  I don't think it created `./data/intro-business-baked`.

File: script/README.md (hunk 2 above)
Comment (edwoodward) [reply script/README.md]: @karenc Change has been made.

File: script/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (helenemccarron) [reply script/README.md]: Karen, the old script was baking the file `book-name-raw.xhtml` to the output `book-name-baked.xhtml`, I believe you changed it when you modified the scripts: https://github.com/openstax/cnx-recipes/commit/3eeeaaf94fc599571b630e9552d0d467ac3b7c86#diff-1b03350bc3912578a35cd7896197b422R53

File: script/README.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+    * `./script/bake-book intro-business`
+6. Test the baked results in the browser
+
+
+PDF Style Development Workflow
+------------------------------
+
+**NOTE**: This workflow assumes the recipe workflow has been completed
+
+1. Move the baked HTML file to the location of the book resources
```
Comment (helenemccarron) [inline script/README.md]: I believe this is no longer necessary with fetching the data using Neb? I thought `neb fetch` and `neb assemble` created the complete html with the images, or `collection.baked.xhtml`? In that case, do we still need to move the file?

File: script/README.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+ * Book name from books.txt file
+
+Example usage
+ * `./script/build-styles intro-business`
+
+
+Recipe Development Workflow
+----------------------------
+
+1. Fetch the data
```
Comment (helenemccarron) [inline script/README.md]: Does this fetch the images?

File: script/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (karenc) [reply script/README.md]: @helenemccarron I've changed it to download to `./data/intro-business/collection.baked.xhtml`, there's no `./data/intro-business-baked/` or `contents/` directories.

File: script/README.md (hunk 6 above)
Comment (karenc) [reply script/README.md]: No, if you leave the files the way they are, the images should work.  (provided images are downloaded when you ran `./script/fetch-html`.)

File: script/README.md (hunk 7 above)
Comment (karenc) [reply script/README.md]: No this doesn't.  Following the way the script used to work, `./script/fetch-html statistics` only gets the html, `./script/fetch-html --with-resources statistics` downlods the html and any resource files.  (May be we should change the README to use `--with-resources`?)

Comment (edwoodward) [reply script/README.md]: Updated

File: script/README.md (hunk 6 above)
Comment (edwoodward) [reply script/README.md]: Updated

**Pull request closed without merging**